e2eAIOK/deltatuner/deltatuner/search/EvolutionarySearchEngine.py

```diff
@@ -0,0 +1,129 @@
+import os
```

Comment:
@tianyil1  here - all of the search engines, base/evolution/sigopt are introduced,  did you modify all of them for LLM compared to what is already in DE-NAS?

Reply:
Yes, all of them have been changed to remove the hardcode part like the domain-specific ops in the previous de-nas part.